### DISCUSSION TITLE
Ignore punctuation marks in searches

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#4359] [Feature Requests]
### Suggest an idea for ExpressionEngine

I have an entry `"Coup d'etat"`. When I search for `"Coup d’etat"` in a relationship field - I can't find the existing entry. It's because of the different symbols `'` and `’`, so `"Coup d’etat" !== "Coup d'etat"`.

Same happens with different quotes.

Can we please make these symbols irrelevant for searches (via relationship field and via tag parameter)?

![Screenshot 2024-06-13 162401](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/dbd933cd-d032-4b32-9acd-67bb2b29e40b)
![Screenshot 2024-06-13 162605](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/f57022b0-34e4-4582-861c-94e057b4e7c5)

**Comment by intoeetive** [comment]
Proposed solution along with some notes: https://github.com/ExpressionEngine/ExpressionEngine/pull/4478

**Comment by TomJaeger** [comment]
This is shipping in 7.5.8
